pkg/metrics/collector.go

```diff
@@ -52,6 +52,7 @@ const (
 	config_audit_title       = "config_audit_title"
 	config_audit_description = "config_audit_description"
 	config_audit_category    = "config_audit_category"
+	config_audit_success     = false
```

Comment:
config_audit_success field name should be equal to `config_audit_success`

Reply:
I've implemented this